backend/src/database/models.rs

```diff
@@ -552,9 +552,20 @@ pub struct Role {
     pub updated_at: NaiveDateTime,
 }
 
+// FromForm defaults bools to false if not present
+#[derive(FromForm)]
+pub struct CreateRoleInput {
+    pub name: String,
+    pub description: Option<String>,
+    pub min_available: i32,
+    pub max_available: i32,
+    pub finalised: bool,
+}
+
```

Comment:
Bit of overlap with CHAOS-31 - might want to use that implementation when it gets merged into main

Reply:
True, will look into this.